evalai/urls.py

```diff
@@ -21,7 +21,7 @@
 from django.conf.urls.static import static
 
 from allauth.account.views import ConfirmEmailView
-from rest_framework_expiring_authtoken.views import obtain_expiring_auth_token
+from rest_framework_simplejwt.views import token_obtain_pair
```

Comment:
Why are we changing this?

Reply:
Previously, we were using `obtain_expiring_auth_token` which returns a user `Token` (**not** `JWT`), `token_obtain_pair` method returns the `Access Token JWT` and `Refresh Token JWT`.